index.html

```diff
@@ -23,6 +23,7 @@
 </head>
 <body>
```

Comment:
Please add unnecessary files and folders like `.vscode` to gitignore. It makes the development environment clean.